libs/native-types/src/mssql.rs

```diff
@@ -1,56 +1,576 @@
-use serde::*;
+use super::TypeParameter;
+use once_cell::sync::Lazy;
+use regex::Regex;
+use serde::{Deserialize, Serialize};
 use serde_json::Value;
+use std::{fmt, io, str::FromStr};
 
-use std::fmt;
+static HEAP_ALLOCATED: Lazy<Vec<MsSqlType>> = Lazy::new(|| {
```

Comment:
What does heap allocated mean? To my understanding these are types that can't be used in conjunction with unique/id/index. Can we indicate that with a better name maybe?

Reply:
This is private. Look into the function in the struct:
https://github.com/prisma/prisma-engines/pull/1348/files/8c8728c0364a82f4c99d73d0d90c03e86fc94197#diff-aa67baeaac2bd89bda9ec50dde477a8b171ffcffce47e91fa1d5a4ba5948211dR127-R131